agent_api/src/main/java/dev/aikido/agent_api/helpers/logging/Logger.java

```diff
@@ -17,6 +19,12 @@ public Logger(Class<?> logClass) {
         this.logLevel = DEFAULT_LOG_LEVEL;
         this.logClass = logClass;
 
+        // We first check "AIKIDO_DEBUG", because "AIKIDO_LOG_LEVEL" takes precedent.
```

Comment:
Please turn this the other way around. AIKIDO_DEBUG should take precedence over anything else as it's likely a temporary flag, where `AIKIDO_LOG_LEVEL` is something you could have in your config at all times.